web/src/main/java/com/zebrunner/reporting/web/util/dozer/ExpirationLocalDateTimeConverter.java

```diff
@@ -14,9 +14,13 @@ public ExpirationLocalDateTimeConverter() {
 
     @Override
     public Integer convertFrom(LocalDateTime source, Integer destination) {
-        ZoneOffset currentZoneOffset = OffsetDateTime.now().getOffset();
-        long numOfSeconds = source.toEpochSecond(currentZoneOffset) - LocalDateTime.now().toEpochSecond(currentZoneOffset);
-        return (int) numOfSeconds;
+        Integer numOfSeconds = null;
+        if (source != null) {
```

Comment:
You accessing instant time (via now) twice and it won't actually be the same (it is a matter of nanoseconds but anyway). Capture instant time once. Also, you can get LDT from ODT by calling `.toLocalDateTime()`